.github/workflows/deploy.yml

```diff
@@ -1,36 +0,0 @@
-name: Deploy
-
-on:
-  push:
-    branches: [master]
-  pull_request:
-    branches: [master]
-
-jobs:      
-  dist:
-    runs-on: ubuntu-latest
-
-    strategy:
-      matrix:
-        platform: [ windows-x64, macos-x64, linux-x64, linux-musl-x64, linux-armv7, linux-arm64 ]
-
-    steps:
-      - name: Cancel prior runs
```

Comment:
Oops, I missed this before, but the previous workflows automatically cancelled runs if a new push occurred while the previous run was executing. I think we should keep that behavior in place.

Reply:
I'm not sure we should continue doing that in this case. If a workflow is running that would result in a new release being created, it could be canceled by a new workflow running off a PR and a new release and Docker image tag would never get created. I think it was more appropriate prior to this PR because the workflows running never resulted in a new release or Docker image tag being created.

Reply:
Okay, that works for me, but if we end up porting over the rest of the tests from CircleCI over to Github actions, I think we'll need to put this back in place.